src/components/CandidateForm.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import styles from "./CandidateForm.module.css";
+import Button from "./Button";
+
+
+const CanditateForm = ({inputsData, sendFormData}) => {
+    const inputs = inputsData;
+
+    const getFormData = () => {
+      const formData = {};
+      document.querySelectorAll('input').forEach((input) => {
+          formData[input.name] = input.value;
+      })
+      sendFormData && sendFormData(formData);
+    }
+
+    return (
+      <form className={styles.form} action="" onSubmit={(e) => {e.preventDefault()}}>
+        {inputs.map((input, index) => (
+          <div className={styles.field} key={index}>
+            <input type="text" name={input.name} placeholder={input.placeholder} defaultValue={input.value} />
```

Comment:
in React we want to have controlled inputs. This means that the value of an input will come from state. and we will change the value by changing the state using onChange event